docs/content/configuration.md

```diff
@@ -832,6 +832,7 @@ included in the actual bundle gzipped tarball.
 | `decision_logs.mask_decision` | `string` | No (default: `system/log/mask`) | Set path of masking decision. |
 | `decision_logs.plugin` | `string` | No | Use the named plugin for decision logging. If this field exists, the other configuration fields are not required. |
 | `decision_logs.console` | `boolean` | No (default: `false`) | Log the decisions locally to the console. When enabled alongside a remote decision logging API the `service` must be configured, the default `service` selection will be disabled. |
+| `decision_logs.nd_builtins_cache` | `boolean` | No (default: `false`) | Enable the non-deterministic builtins caching system during policy evaluation, and include the contents of the cache in decision logs. Note that decision logs that are larger than `upload_size_limit_bytes` will silently drop the `nd_builtins_cache` key from the log entry before uploading. |
```

Comment:
This one configuration flag sets off everything ND builtins cache-related. In the future, it could be worthwhile to allow enabling the cache *without* requiring the `decision_logs` plugin's involvement. :thinking:

Reply:
I was expecting the `nd_builtins_cache` option as a top level object in the config with options to set it in the decision logs etc. The decision logger's `Log` would simply add this if it was set in the input object. I think keeping this as its own config would make it easier for us to update with additional extensions.

Reply:
If that's what we envision, let's promote it to top-level.